### DISCUSSION TITLE
Account is approved for education pack but no benefits yet.

### DISCUSSION THREAD

**Discussion by AshishKumar9532** [community/community#63639] [GitHub Education] [answered]
### Hi everyone,

***Hello everyone,***
I have been granted the student benefits, but I have not received any email notification.
Approved Information: Congrats! Your request for [@AshishKumar9532](https://github.com/AshishKumar9532) was approved and your benefits will be available soon.

how long i have to wait??

![Screenshot (182)](https://github.com/community/community/assets/99255937/5e35ebe5-e71d-4fdb-9c75-1472da3b329a)

**Comment by CJKennedy00** [comment, marked as answer]
It may take a few days before you'll be able to access your benefits. I got mine 4 business days after approval.

**Reply by AshishKumar9532** [reply]
damn, previously it took few seconds only.

**Reply by brownjeesus** [reply]
my developer pack was approved too yesterday but still no benefits recieved from there end

**Comment by Well300O** [comment]
I'm facing same issue, account is approved but no benefits in account.

**Comment by AshishKumar9532** [comment]
It has been  about 72 hours but no benefits

**Comment by AshishKumar9532** [comment]
Got it now,after 78 hours
